packages/data-service-generator/src/admin/entity/entity-list-component/entity-list-component.template.tsx

```diff
@@ -1,20 +1,14 @@
 import * as React from "react";
-import { List, Datagrid, ListProps } from "react-admin";
+import { List, Datagrid } from "react-admin";
 //@ts-ignore
 import Pagination from "../Components/Pagination";
 
 declare const ENTITY_PLURAL_DISPLAY_NAME: string;
 declare const CELLS: React.ReactElement[];
 
-export const ENTITY_LIST = (props: ListProps): React.ReactElement => {
+export const ENTITY_LIST = (): React.ReactElement => {
   return (
-    <List
-      {...props}
-      bulkActionButtons={false}
-      title={ENTITY_PLURAL_DISPLAY_NAME}
-      perPage={50}
-      pagination={<Pagination />}
-    >
+    <List>
```

Comment:
Are the list props no longer needed ?

Reply:
`bulkActionButtons` is deprecated. we can have rest of props.

Reply:
I saw in pagination component, we extending rowsPerPageOptions, so I revert that change

Reply:
> Are the list props no longer needed ?

yes